### DISCUSSION TITLE
Could be possible to specify the name of output encrypted/decrypted file instead of letting Kryptor set it by default when doing -e/-d -p??

### DISCUSSION THREAD

**Discussion by q2dg** [samuel-lucas6/Kryptor#77] [Ideas]
Thanks!

PD: I don't know if this could be related, but the feature of show in stdout the result of the decryption/encryption instead of going to a predefined file by default would be kind, too

**Comment by samuel-lucas6** [comment]
Hi, this is a duplicate of #75.
